cratedb_jdbc/manifest.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='utf-8' ?>
 
-<connector-plugin class='cratedb_jdbc' superclass='jdbc' plugin-version='0.0.1' name='CreateDB' version='18.1' min-version-tableau='2020.4'>
+<connector-plugin class='cratedb_jdbc' superclass='jdbc' plugin-version='0.0.1' name='CrateDB' version='18.1' min-version-tableau='2020.4'>

```

Comment:
This is how it's displayed 
![image](https://github.com/crate/tableau-connector/assets/37985796/a157fed5-f7d8-4eef-aa10-84dbb57088f2)

@ckurze

That ok?

Reply:
@surister Sorry for the late response - Can we just have CrateDB and remove the "by Crate.io", please?

Thx.

Reply:
Sure thing!